app/controllers/v0/sessions_controller.rb

```diff
@@ -122,7 +122,7 @@ def saml_callback
         StatsD.increment(STATSD_LOGIN_NEW_USER_KEY) if @sso_service.new_login?
         StatsD.increment(STATSD_SSO_CALLBACK_KEY, tags: ['status:success', "context:#{context_key}"])
       else
-        redirect_to Settings.saml.relay + "?auth=fail&code=#{@sso_service.auth_error_code}"
+        redirect_to Settings.saml.relay.original + "?auth=fail&code=#{@sso_service.auth_error_code}"
```

Comment:
this name "original" is a little confusing / misleading to me because we won't have the same context in the future. Is there a better name we could use?

Reply:
much more agreeable